pkg/store/change/change_test.go

```diff
@@ -102,11 +100,11 @@ func Test_ConfigValue(t *testing.T) {
 	path := "/cont1a/cont2a/leaf2a"
 	value := 13
 
-	configValue2a, _ := NewChangeValue(path, types.NewTypedValueUint64(uint(value)), false)
+	configValue2a, _ := types.NewChangeValue(path, types.NewTypedValueUint64(uint(value)), false)
 
 	assert.Equal(t, configValue2a.Path, path)
 
-	assert.Equal(t, configValue2a.String(), "/cont1a/cont2a/leaf2a [13 0 0 0 0 0 0 0] false")
+	assert.Equal(t, configValue2a.String(), `path:"/cont1a/cont2a/leaf2a" value:<bytes:"\r\000\000\000\000\000\000\000" type:UINT > `)
```

Comment:
I think it's better if we do not compare strings directly but go in depth and compare the path and the value as objects

Reply:
done